cmd/adsysd/client/policy.go

```diff
@@ -122,6 +122,33 @@ func (a *App) installPolicy() {
 	policyCmd.AddCommand(updateCmd)
 	cmdhandler.RegisterAlias(updateCmd, &a.rootCmd)
 
+	var purgeMachine, purgeAll *bool
+	purgeCmd := &cobra.Command{
+		Use:   "purge [USER_NAME]",
+		Short: i18n.G("Purges policies for the current user or a specified one"),
+		Args:  cmdhandler.ZeroOrNArgs(1),
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			// All and machine options don’t take arguments
+			if *purgeAll || *purgeMachine || len(args) != 0 {
+				return nil, cobra.ShellCompDirectiveNoFileComp
+			}
+
+			return a.completeWithConnectedUsers()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var user string
+			if len(args) > 0 {
+				user = args[0]
+			}
+			return a.purge(*purgeMachine, *purgeAll, user)
+		},
+	}
+	purgeMachine = purgeCmd.Flags().BoolP("machine", "m", false, i18n.G("machine purges the policy of the computer."))
+	purgeAll = purgeCmd.Flags().BoolP("all", "a", false, i18n.G("all purges the policy of the computer and all the logged in users. -m or USER_NAME cannot be used with this option."))
+	purgeCmd.MarkFlagsMutuallyExclusive("machine", "all")
+	policyCmd.AddCommand(purgeCmd)
+	cmdhandler.RegisterAlias(purgeCmd, &a.rootCmd)
```

Comment:
as this is a "dangerous" command, I wouldn’t register an alias for this one and only keep it under the "policy" subcommand.